docs/_sources/user/group_mdmr.rst

```diff
@@ -38,15 +38,10 @@ Configuration Using a YAML File
 """"""""""""""""""""""""""""""""
 To configure CWAS options within a YAML file, add the following lines to your file (with appropriate substitutions for paths):
  
-.. code-block:: yaml
-
-    runCWAS = [0]
-    cwas_roi_file = '/path/to/cwas_mask_file'
-    cwas_regressor_file = '/path/to/cwas_regressor_file'
-    cwas_regressor_participant_column = 'ID
-    cwas_regressor_columns = ['FIQ', 'VIQ', 'PIQ']
-    cwas_permutations = 500
-    cwas_parallel_nodes = 3
+.. literalinclude:: /references/group_config_template.yml
+   :language: YAML
+   :start-at: mdmr:
+   :end-before: isc_isfc:
```

Comment:
I think we want the comment above `mdmr:` and not the one above `isc_isfc:` included in this snippet?

( https://github.com/FCP-INDI/C-PAC/blob/5b2465fcc8a9986462db774046edbe55d4b91f7e/CPAC/resources/configs/group_config_template.yml#L255-L285 instead of https://github.com/FCP-INDI/C-PAC/blob/5b2465fcc8a9986462db774046edbe55d4b91f7e/CPAC/resources/configs/group_config_template.yml#L256-L286 )

```suggestion
   :start-at: # Multivariate Distance Matrix Regression (MDMR)
   :end-before: # Inter-Subject Correlation (ISC) & Inter-Subject Functional Correlation (ISFC)
```

Reply:
Resolved in 387f71f8f53b24bef1479cf778b90b475b27a5ee:

https://github.com/FCP-INDI/fcp-indi.github.io/blob/387f71f8f53b24bef1479cf778b90b475b27a5ee/docs/_sources/user/group_mdmr.rst?plain=1#L41-L44

![fcp-indi.github.io/docs/nightly/user/group_mdmr#configuration-using-a-yaml-file](https://user-images.githubusercontent.com/5974438/223251897-0d9a7417-3a7c-401f-bc91-50a01fdaedbc.png)